infra/managed-cluster/aks-cluster.bicep

```diff
@@ -147,12 +149,12 @@ module deployAKS 'br/SharedDefraRegistry:container-service.managed-clusters:0.5.
     aksClusterNetworkPlugin: 'azure'
     aksClusterNetworkPluginMode: 'overlay'
     aksClusterNetworkPolicy: 'calico'
-    aksClusterPodCidr: '172.16.0.0/16'
-    aksClusterServiceCidr: '172.18.0.0/16'
+    aksClusterPodCidr: cluster.podCidr
+    aksClusterServiceCidr: cluster.serviceCidr
     aksClusterDnsServiceIP: '172.18.255.250'
```

Comment:
where did this come from? but worthwhile wacking in common?

Reply:
This is from aks baseline. I will move it to vars.